### DISCUSSION TITLE
12/13 정민채 블로그 2회차

### DISCUSSION THREAD

**Discussion by Jungminchae** [Orm-camp/blog-feedback#52] [동글동글 4기 피드백]
https://chaechae.life/blog/fastapi-password-hasing-with-agron2

감사합니다

**Comment by smahn9123** [comment]
나중에 비밀번호 해싱할 때 argon2 고려해보겠습니다. 유용한 내용 감사합니다.

**Comment by devSeyong** [comment]
아직 모르는 내용이 많아서 잘 이해는 못하겠지만 글의 문단이 길지 않고 한눈에 들어와서 좋습니다!
